components/atoms/Accordion/Accordion.js

```diff
@@ -0,0 +1,55 @@
+import propTypes from "prop-types";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+
+export function Accordion({ title, summary, description }) {
+  const [openFlag, setOpenFlag] = useState(false);
+  function openSummary() {
+    setOpenFlag(!openFlag);
+  }
+
+  return (
+    <div className="border-gray-300 flex border border-b-2 border-r-2 mb-2 p-8 shadow-lg">
+      <div className="w-8  min-w-8 relative top-5">
+        {openFlag ? (
+          <FontAwesomeIcon
+            icon={faChevronRight}
+            rotation={90}
+            size="2x"
+            color="red"
+          />
+        ) : (
+          <FontAwesomeIcon icon={faChevronRight} size="2x" color="red" />
+        )}
+      </div>
+
+      <details className="flex-1 flex-wrap">
+        <summary onClick={() => openSummary()}>
+          <h3>{title}</h3>
+          {summary}
```

Comment:
We should probably put this in a span or some other container element so that we can style it if we need to.